modules/telco-core-rds-container.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/core/telco-core-ref-crs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-core-rds-container_{context}"]
+= Extracting the telco core reference design configuration CRs
+
+You can extract the complete set of custom resources (CRs) for the telco core profile from the `telco-core-rds-rhel9` container image. 
+The container image has both the required CRs, and the optional CRs, for the telco core profile.
+
+.Prerequisites
+
+* You have installed `podman`.
+
+.Procedure
+
+* Extract the content from the `telco-core-rds-rhel9` container image by running the following commands:
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal]
+----
+$ podman run -it registry-proxy.engineering.redhat.com/rh-osbs/openshift4-telco-core-rds-rhel9:v4.16.0-1 | base64 -d | tar xv -C out
```

Comment:
I don't think so yet, because 4.16 is still pre-ga.
but I'd expect it to be something like this:
registry.redhat.io/openshift4/telco-core-rds-rhel9:v4.16
This needs to be verified before being published unless someone else is sure and can confirm.

Reply:
@shajmakh - the link has been updated. It won't go live until the GA date so we're running with this for now and will double-check at GA. Let me know if any other issues or if you can give your LGTM. I need to close this out this week. Thanks!

Reply:
thanks for addressing this. sounds good to me.